src/providers/user-data/user-data.ts

```diff
@@ -30,6 +30,10 @@ export class UserDataProvider {
   public onUpdateWallet$: Subject<Wallet> = new Subject();
   public onSelectProfile$: Subject<Profile> = new Subject();
 
+  public get isTestNet(): boolean {
```

Comment:
A few things regarding this. I would calling it "isDevnet". You can also tweak the code to match in the mainnet-only directive (and change that code to call `isDevnet` too. From the directive:

```
    if (this.userDataProvider.currentNetwork.type === NetworkType.Devnet) {
      this.elementRef.nativeElement.style.display = 'none';
    }
```

No need for the constant once you use this